src/backend/crew_ai/library_context/dynamic_context.py

```
@@ -38,23 +48,55 @@ def __init__(self, library_name: str):
+    def _load_from_pregenerated(self) -> Optional[Dict]:
+        """
+        Try to load documentation from pre-generated JSON file.
+        
+        Returns:
+            Dictionary containing library metadata and keywords, or None if not found
+        """
+        libdoc_file = LIBDOCS_DIR / f"{self.library_name.lower()}.json"
+        
+        if libdoc_file.exists():
+            try:
+                with open(libdoc_file, 'r', encoding='utf-8') as f:
```

Ruff (0.15.5)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)